<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>771</width>
      <height>431</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>HOME PAGE</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GPS</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ACC</string>
      </property>
     </widget>
     <widget class="QPushButton" name="GPS_begin_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>BEGIN</string>
      </property>
      <property name="iconSize">
       <size>
        <width>25</width>
        <height>22</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="GPS_stop_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ACC_begin_button">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>BEGIN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ACC_stop_button">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>STOP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SDÂç°</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE</string>
      </property>
     </widget>
     <widget class="QLabel" name="base_time_label">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Time_base:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>GPS</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Latitude:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Longitude:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Speed:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Visible satellite:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UTC Date:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>260</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>UTC Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_latitude">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>60</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_longitude">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_speed">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>140</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_satellite_num">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>180</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_UTCdate">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>220</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_UTCtime">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>260</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Accuracy:</string>
      </property>
     </widget>
     <widget class="QLabel" name="GPS_accuracy">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>300</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>AR PL UKai CN</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color:rgb(255, 0, 0)</string>
      </property>
      <property name="text">
       <string>NaN</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>ACC</string>
     </attribute>
     <widget class="QCustomPlot" name="ACC_plot" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>761</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
